modules/analyzing-all-violations.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operating/respond-to-violations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="analyzing-all-violations_{context}"]
+= Analyzing all violations
+
+By viewing the *Violations* page, you can analyze all violations and take corrective action.
+
+.Procedure
+
+. In the {product-title-short} portal, click *Violations*.
+. Choose the appropriate method to view the violations from the drop-down list, which is in the upper left of the page:
+** To display the findings for application workloads, select *Applications view*
+** To display the findings for platform components in OpenShift, select *Platform view*.
+** To display the findings for application workloads and platform components simultaneously, select *Full view*.
```

Comment:
I believe the `Full view` includes everything. You can think of it like this:

`Application workloads` + `Platform components in OpenShift` + `[whatever isn't in the previous 2 categories]` = `Full view`

Those other violations could be for Cluster role bindings, Secrets, etc. I'll ping Charmik and let him explain the full list.

Reply:
I believe Charmik's answer to a different comment explains what can show up in that 3rd set:

> For other resources violations, the type will be the resource type: Secrets, Configmaps, Cluster_roles, Cluster_role_bindings, Network_policies, Security_context_constraints, Egress_firewalls

Reply:
Thanks for suggesting, however, I would recommend that we do not change the text to match the UI descriptions, unless the UX team updates the descriptions.
![Screenshot from 2024-11-06 12-39-55](https://github.com/user-attachments/assets/f72fe54b-87de-431e-addb-d79c83ae0000)